github/event_types.go

```diff
@@ -165,6 +165,28 @@ type ProjectColumnChange struct {
 	} `json:"name,omitempty"`
 }
 
+// TeamChange represents the changes when a team has been edited.
+type TeamChange struct {
+	Description *struct {
+		From *string `json:"from,omitempty"`
+	} `json:"description,omitempty"`
+	Name *struct {
+		From *string `json:"from,omitempty"`
+	} `json:"name,omitempty"`
+	Privacy *struct {
+		From *string `json:"from,omitempty"`
+	} `json:"privacy,omitempty"`
+	Repository *struct {
+		Permissions *struct {
+			From *struct {
+				Admin *string `json:"admin,omitempty"`
+				Push  *string `json:"push,omitempty"`
+				Pull  *string `json:"pull,omitempty"`
```

Comment:
This is _really_ minor, but the order in the documentation is `admin`, then `pull`, then `push`. Is there a specific reason you chose to swap the order of Push and Pull? If not, maybe it'd be better to stay consistent with the docs by swapping the two.